cmd/clusterctl/examples/tools/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM debian:stretch-slim
```

Comment:
Hi @andrewsykim,

Please remind me to give you creds to the CAPV GCS project so you can get this image pushed there.